Panic when event object has null firstTimestamp property

panic: interface conversion: interface {} is nil, not string

goroutine 5 [running]:
github.com/stepro/kudo/pkg/pod.track.func1(0xc000242000, 0x360)
	/Users/stephpr/src/kudo/pkg/pod/pod.go:25 +0x3de
github.com/stepro/kudo/pkg/output.(*lineWriter).Write(0xc000012030, 0xc000122000, 0x361, 0x8000, 0x361, 0x0, 0x0)
	/Users/stephpr/src/kudo/pkg/output/lineWriter.go:42 +0x20c
io.copyBuffer(0x17b0008, 0xc000012030, 0x12c4480, 0xc000010018, 0xc000122000, 0x8000, 0x8000, 0x11eb941, 0xc0000c42c0, 0xc000092d80)
	/usr/local/go/src/io/io.go:407 +0x1fb
io.Copy(...)
	/usr/local/go/src/io/io.go:364
os/exec.(*Cmd).writerDescriptor.func1(0x0, 0x0)
	/usr/local/go/src/os/exec/exec.go:311 +0x63
os/exec.(*Cmd).Start.func1(0xc0000c42c0, 0xc00000e080)
	/usr/local/go/src/os/exec/exec.go:441 +0x27
created by os/exec.(*Cmd).Start
	/usr/local/go/src/os/exec/exec.go:440 +0x632

--- a/pkg/pod/pod.go
+++ b/pkg/pod/pod.go
@@ -22,7 +22,7 @@ func track(k *kubectl.CLI, pod string, op output.Operation) func() {
 			return
 		}
 		obj := event["object"].(map[string]interface{})
-		firstTimestamp := obj["firstTimestamp"].(string)
+		firstTimestamp, _ := obj["firstTimestamp"].(string)
 		if firstTimestamp < timestamp {
 			return
 		}

